classes/Currency.php

```diff
@@ -239,10 +257,11 @@ public function __construct($id = null, $idLang = null, $idShop = null)
              * the field as a string.
              */
             if (is_array($this->symbol)) {
-                $this->localizedSymbols = $this->symbol;
+
+                $this->localizedSymbol = $this->symbol;
```

Comment:
This is a mismatch, 1.7.6.x and develop have the same property but one adds a "s" and the other one does not

DONT MERGE BEFORE FIX

Reply:
OK, I confirm the same property has been added on `1.7.6.x` and `develop` by 2 different devs 😭 
- https://github.com/PrestaShop/PrestaShop/pull/14972/files#diff-bd916bf385a432f6bf6f9b61421ad52bR124
- https://github.com/PrestaShop/PrestaShop/commit/72a74d88bdd896b3e9f5248c7b050fce6f2cee21#diff-bd916bf385a432f6bf6f9b61421ad52bR108

Ping @atomiix and @jolelievre